io.github.jchempaint.JChemPaint.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.jchempaint.JChemPaint
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk8
+command: jchempaint
+
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+  - --env=PATH=/app/jre/bin:/app/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+  - --filesystem=home
```

Comment:
defaulting to home seems a bit too much, can this be limited to a few select directories?

Reply:
No, the dialog defaults to $HOME and there is no XDG directory for both images and molecular structure data.

Reply:
unfortunate.